core/src/main/java/org/testcontainers/dockerclient/TestcontainersHostPropertyClientProviderStrategy.java

```diff
@@ -46,6 +48,14 @@ public TransportConfig getTransportConfig() throws InvalidConfigurationException
             .build();
     }
 
+    @Override
+    protected boolean isApplicable() {
+        return (
+            TestcontainersConfiguration.getInstance().getUserProperty("tc.host", null) != null &&
+            this.dockerClientConfig != null
```

Comment:
With the changes in the constructor, isn't this line enough and we can omit the duplicating check for `tc.host`?

Reply:
Fixed